### PR TITLE
Add com.github.Rosalie241.RMG

### PR DESCRIPTION
**Pull request by Rosalie241**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Thanks to @JakobDev for all the help with this, I hope I did everything right :heart:

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.Rosalie241.RMG

**Comment by Rosalie241** [comment]
bot, build com.github.Rosalie241.RMG

**Comment by JakobDev** [comment]
Looks like I have a Bug in jdAppdataEdit. I  used the project license list also for the metadata licenses, this is why there is a invalid metadata license. Will be fixed in the next release.

Just 2 things about the finish args that I forgot:
1. You should replace --device=dri with --device=all to make Gamepads working.
2. If RMG supports Wayland ('m a Nvidia user, so I don't know), you should replace --socket=x11 with --socket=fallback-x11 and --socket=wayland.

**Comment by Rosalie241** [comment]
> Looks like I have a Bug in jdAppdataEdit. I used the project license list also for the metadata licenses, this is why there is a invalid metadata license. Will be fixed in the next release.

:heart:

> Just 2 things about the finish args that I forgot:
> 
>     1. You should replace --device=dri with --device=all to make Gamepads working.

Good point, thank you :heart:

>     2. If RMG supports Wayland (I'm a Nvidia user, so I don't know), you should replace --socket=x11 with --socket=fallback-x11 and --socket=wayland.

it doesn't support wayland yet.

**Comment by Rosalie241** [comment]
bot, build com.github.Rosalie241.RMG

File: minizip.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (barthalion) [inline minizip.json:1]: Any reason this is a separate file and not part of the main manifest?

Comment (Rosalie241) [reply minizip.json:1]: I don't think there is a specific reason but @JakobDev made it, though I can try to put this in the main manifest

**Comment by Rosalie241** [comment]
bot, build com.github.Rosalie241.RMG

**Comment by Rosalie241** [comment]
bot, build com.github.Rosalie241.RMG

**Pull request closed without merging**